.github/scripts/generate_pr_description_zhipu.py

```
@@ -0,0 +1,81 @@
+
+        # Retrieve the API key from the environment variable set by the GitHub Action
+        api_key = os.getenv("ZHIPU_API_KEY")
+        if not api_key:
+            raise ValueError("ZHIPU_API_KEY environment variable is not set.")
+
+        client = ZhipuAI(api_key=api_key)
+
+        # Truncate diff to prevent token limit issues (optional, but good practice)
+        MAX_DIFF_LENGTH = 50000
+        if len(diff_content) > MAX_DIFF_LENGTH:
+            logger.info(f"Diff content truncated from {len(diff_content)} to {MAX_DIFF_LENGTH} characters.")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)